docs/05-technical-reference/00-architecture/obsv-04-architecture-kiali.md

```diff
@@ -6,10 +6,9 @@ The following diagram presents the overall Kiali architecture and the way the co
 
 ![Kiali architecture](./assets/obsv-kiali-architecture.svg)
 
-1. Use the Kyma Console or direct URL to access Kiali.
-2. To ensure authentication, the Keycloak Gatekeeper checks if you have a valid token.
-3. If not, the Keycloak Gatekeeper redirects you to the identity provider to log in.
-4. After a successful log in, you can access the Kiali service which serves the website. The service exposes an endpoint and acts as an entry point for the Kiali deployment.
-5. Kiali collects the information on the cluster health from the following sources:
-   * API server which provides data on the cluster state.
-   * Service Mesh by analyzing metrics Prometheus scrapes from the Istio Pod.
+1. Use the Kyma Console or direct URL to access Kiali. Learn more about [accessing Kiali](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md).
+2. Kiali collects the information on the cluster health from the following sources:
+   * Jaeger tracing
+   * Kubernetes API server, which provides data on the cluster state
+   * Service Mesh using Prometheus
```

Comment:
Please move the last item to the first point in the list as it is most important.

Maybe it is worth to say that it is the metrics data coming from istio collected by prometheus

Reply:
done with new commit